lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -690,14 +727,21 @@ export abstract class ClientApplication {
      * @param account 
      */
     setActiveAccount(account: AccountInfo | null): void {
-        this.activeLocalAccountId = account ? account.localAccountId : null;
+        if (account) {
+            this.logger.verbose("Active account set");
+            this.activeLocalAccountId = account.localAccountId;
+        } else {
+            this.logger.verbose("No account passed, active account not set");
+            this.activeLocalAccountId = null;
+        }
     }
 
     /**
      * Gets the currently active account
      */
     getActiveAccount(): AccountInfo | null {
         if (!this.activeLocalAccountId) {
+            this.logger.verbose("No active account");
```

Comment:
```suggestion
            this.logger.verbose("getActiveAccount: No active account");
```

Reply:
I think prefixing log messages like so can help make it easier to follow.

Reply:
Makes sense, added